docs/meadowlark-software-upgrade.md

```diff
@@ -0,0 +1,8 @@
+# Upgrade Meadownlark PC software version
+
+To upgrade your "PC software version" use these steps:
+
+- From "Add and remove programs", remove CellDrive5000 and "National Instruments Software".
+- From "Device manager", open the "Meadowlark Optics" group, right click `mlousb`, click "Uninstall device", check "Delete the driver software for this device", and click "Uninstall". Uninstall `Meadowlark Optics D5020 LC Driver` following the same steps.
+- Using the USB stick provided by Meadowlark, reinstall the USB drivers and CellDrive5000. 
+- **Upgrading users:** you will need to reinstall the Meadowlark device to your micromanager configuration file, because the device driver's name has changed to `MeadowlarkLC`. 
```

Comment:
Does this belong here? In `docs/microscope-installation-guide.md` you say that the device is called `MeadowlarkLcOpenSource`. At what point did we rename it to `MeadowlarkLC`? Also, this is more general MM setup rather than related to upgrading the PC software version, I think

Reply:
P.S. These are instruction for users upgrading from recOrder 0.3 to recOrder 0.4, I think we can delete the Upgrading users section from here. Or you may wish to leave a comment if anyone wants to downgrade (very unlikely)?

Reply:
Great calls @ieivanov. I've:
- moved the "PC software version instructions" into the installation guide
- removed the upgrading instructions from here, because they're not needed